larrecodnn/ImagePatternAlgs/Tensorflow/Modules/EmTrackMichelIdTl_module.cc

```diff
@@ -1,11 +1,10 @@
 /////////////////////////////////////////////////////////////////////////////////////
 // Class:       EmTrackMichelIdTl
 // Module Type: producer
-// File:        EmTrackMichelIdTl_module.cc (tool interface ver. of EmTrackMichelId)
+// File:        EmTrackMichelIdTl_module.cc
 // Authors:     D.Stefan (dorota.stefan@cern.ch), from DUNE, CERN/NCBJ
 //              P.Plonski (pplonski86@gmail.com), from DUNE, WUT
 //              R.Sulej (robert.sulej@cern.ch),   from DUNE, FNAL/NCBJ
-//              M.Wang (mwang@fnal.gov),          from DUNE, FNAL, tool interface ver
```

Comment:
It would be good to keep Mike's name in the list of authors.

Reply:
Oh yes of course, that must be a mistake. I will fix it.